### DISCUSSION TITLE
Special Announcement#2:  与月亮计划的第二次沟通

### DISCUSSION THREAD

**Discussion by N071** [LocalizeLimbusCompany/LocalizeLimbusCompany#165] [公告 | Announcements]
Although most of the users of this project uses CN as their primary language, EN will be used here since we're on GitHub.
Our team has planned a visit to PM's office on 8/17, and here's all the things going on regarding this whole plan.
中文版本的此公告：[BiliBili专栏](https://www.bilibili.com/read/cv26005609)

**Comment by N071** [comment]
23/8/15, we sent an e-mail prior to our planned visit, acknowledging the plan and requesting confirmation.
## Original screenshot
<details>

![7b008978c137222bdb7e0dd527f9c8b0](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/128023427/65016f8f-093d-4cee-8683-c83c9464e244)
</details>

## KR Raw
<details>

존경하는 프로젝트 문에게:

안녕하십니까.저희는 이미 림버스 컴퍼니의 중국어 번역을 반년간 진행한 LLC 입니다.이번엔 중국에서 직접 한국으로 대표를 파견하여 프로젝트 문을 방문하려고 이편지를 올렸습니다.

혹시나 contact 이메일이 통하지 못할 만일의 경우를 대비해 business 이메일에도 편지를 보냈습니다.프로젝트 문에게 불편함을 드려서 미리 사과드리겠습니다.그러나 이번에 관련되는 사항은 business 라고 말해도 문제가 없을 것이라고 생각합니다.

8.17(목요일)에 프로젝트 문을 방문해볼 계획입니다.혹시나 프로젝트 문이 많이 불편하다고 생각되면 프로젝트 문의 생각을 저희에게 알려드리기를 바랍니다.(중국에서 한번 찾아가본다는것은 생각보다 어려운 일이기에,저희의 방문을 거절하지는 말았으면 좋겠습니다)

간단히 말해드리자면 방문의 목적은 프로젝트 문에게 저희 모듈 및 성과를 보여주고 림버스 컴퍼니의 중국어 공식적 도입을 위해 협상을 진행하는 것입니다.더욱 상세한 사항은 8.17에 알려드리겠습니다.

만약 8.17전 까지 이편지에 대한 회답이 없으면 그냥 프로젝트 문에게 다른 의견이 없다는 것으로 여기고 계획대로 방문할 예정입니다.마지막으로 프로젝트문의 사업이 잘되여 나가기를 축원합니다.

2023.08.15
LLC 올림
</details>

## Polished translation by DeepL to CN
<details>

亲爱的月亮计划

您好！我们是LLC，已经为Limbus Company的中文翻译工作了半年。

我也把信发到了您的 BUSINESS 邮箱，以防联系邮箱打不通。我对给月亮计划带来的不便提前表示歉意，但我想此事应当也算是属于 BUSINESS 的范畴。

我们计划于 8 月 17 日（周四）访问月亮计划的办公室，如果您觉得不方便，希望您能告诉我们您的想法（在中国访问比您想象的要困难得多，所以我们希望您不要拒绝我们的访问）。

总之，此次访问的目的是向月亮计划展示我们的模块和成果，并商谈将LLC正式化的事宜。

如果在 8 月 17 日之前没有回信，我们便认为您没有其他意见，并将按计划进行访问。

2023 年 8 月 15 日
发件人：LLC
</details>


By the way, our abbreviation is LLC, which stands for "Localize Limbus Company". Sometimes the translator misunderstands LLC as "Limited Liability Company" and produces a weird outcome.
If you plan to use machine translation, please note this issue.

**Comment by N071** [comment]
23/8/16, we have received a response.

## Original screenshot
<details>

![d81e3c3633297ef73f29bfa0f6d439da](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/128023427/39b762c4-f704-47b3-b3fc-f8b26027daf3)
![7534cf4e65612380fc5099d178e05267](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/128023427/9f9aaebf-7b78-433c-aef1-43c1ff36f9ef)
</details>

## KR Raw
<details>
안녕하세요. LCC님.
처음 인사 드립니다. 저는 프로젝트문 경영지원팀 소속 유현수라고 합니다.
메일 확인 후 답변드리고자 첫 인사를 메일로 보내게 된 점 양해 부탁드립니다.

직접 번역을 자처해 보다 많은 중국 게이머분들이 쾌적하게 게임 진행을 하고자하는 귀하의 선의와 자사 게임에 대한 뜨거운 성원이 느껴져 너무나 감사한 마음입니다.
공식적인 번역 계약을 위해 직접 방문을 원하신다 말씀주어 너무나 감사한 마음이나 방문은 어려울 것 같다 미리 말씀드립니다.

현재 중국 공식 서비스를 준비하기에는 현재 내부적으로 여유가 턱 없이 부족해 어려운 상황입니다.
그리고 중국의 게임관련법에 맞춰 다시 대응하기에 시간이 많이 소요되며 이전 작과 다르게 Limbus Company의 경우 라이브 서비스 중으로 중국 판호 등을 위해 법적 자문 및 문서 작업들이 지속적으로 필요할 것이다 사료됩니다.
하여 중국 공식 서비스는 현재로서 굉장히 어려운 상황임을 감안해주시면 감사하겠습니다.

지속적인 관심과 번역 작업은 너무나 감사한 마음이오나 방문은 어려운 점 다시 한번 양해 부탁드립니다. 

감사합니다.
유현수 드림.


<이하 번역기를 통해 내용 전달드리며 내용의 오류가 있을 수 있음을 양해 바랍니다.>
<请注意，以下内容由翻译机员翻译，可能存在错误>

你好。LCC
我想自我介绍一下 我是项目管理支持团队的 Hyunsoo Yoo。
请您理解，我通过电子邮件向您致以第一声问候，以便在查看您的电子邮件后给您回复。

我们非常感谢您对我们游戏的良苦用心和热情支持，也非常感谢您愿意亲自翻译我们的游戏，让更多的中国玩家享受到游戏的乐趣。
我们非常感谢你愿意亲自来我们公司签订正式的翻译协议，但我们很遗憾地告诉你，这是不可能的。

目前，我们没有足够的内部资源来准备官方中文服务。
此外，重新应对中国游戏相关法律需要大量时间，与往年不同的是，Limbus 公司正处于实服过程中，中方审批需要不断进行法律咨询和文件准备工作。
因此，我们希望您能考虑到官方中服目前正处于非常困难的境地。

我们非常感谢您一直以来的关心和翻译工作，但很遗憾我们无法拜访您。

谢谢。
Yoo Hyunsoo.
</details>

By the way, our abbreviation is LLC, which stands for "Localize Limbus Company". Sometimes the translator misunderstands LLC as "Limited Liability Company" and produces a weird outcome.
If you plan to use machine translation, please note this issue.

## Our reaction and comment on this response
We think that Project Moon has misunderstood our intention. What we want is more acknowledgement and potential help (which we planned to talk more about on the meeting), not a straight-away commercial request of a business contract.
We know that we are a community-driven organization, and we will never make such an arrogant request.

Though I think that we can definitely make our intention clearer in the prior email, but anyway, PM (indirectly and euphemistically) rejected our request of a meeting.
Thus, we cancelled the planned visit on the next day, and the team member left Korea in a couple of days.

On top of this, unfortunately, the team member had a serious fever as he got home, unable to work, stalling our further response.
But thanks to him, we now received another official e-mail from ProjMoon, on top of the [previous one](https://github.com/orgs/LocalizeLimbusCompany/discussions/70). This is already a great success, indicating the possibility of more communications in the future.
As he has recovered, and is back online now, we're planning a follow-up response to fix the misunderstanding.

**Comment by N071** [comment]
@1Knightey1
Thought you would be interested in this topic.

**Reply by 1Knightey1** [reply]
Oh, thank you!
It's sad that you couldn't meet PMoon in person in the end, but, considering the last visit of "fans", it might be for the better.
Going to root for you and hope that PMoon will answer to your next letters!
